tests/test_institution.rb

```diff
@@ -12,7 +12,7 @@ def setup
       client = NordigenClient.new(secret_id: ENV['SECRET_ID'], secret_key: ENV['SECRET_KEY'])
       client.generate_token
       @institution = InstitutionsApi.new(client)
-      @institution_id = 'REVOLUT_REVOGB21'
+      @institution_id = 'REVOLUT_REVOLT21'
```

Comment:
curious why you changed this? :D

Reply:
It no longer seems to exist (the spec failed originally)